src/protocol/iobuf.c

```diff
@@ -14,16 +14,17 @@
 
 #include <assert.h>
 #include <string.h>
+#include "zenoh-pico/config.h"
 #include "zenoh-pico/protocol/iobuf.h"
 
 /*------------------ IOSli ------------------*/
```

Comment:
I suggest to not change the semantic of the `_z_iosli_wrap` to be readonly.

Reply:
done.